test/e2e/sgx/sgx.go

```diff
@@ -113,4 +115,32 @@ func describe() {
 		ginkgo.By("waiting the pod to finish successfully")
 		e2epod.NewPodClient(f).WaitForSuccess(pod.ObjectMeta.Name, 60*time.Second)
 	})
+
+	ginkgo.It("deploys SGX plugin with operator", func() {
+		utils.Kubectl("", "apply", "-k", "deployments/operator/default/kustomization.yaml")
+
+		if _, err := e2epod.WaitForPodsWithLabelRunningReady(f.ClientSet, ns, labels.Set{"control-plane": "controller-manager"}.AsSelector(), 1, timeout); err != nil {
+			framework.Failf("unable to wait for all pods to be running and ready: %v", err)
+		}
+
+		utils.Kubectl("", "apply", "-f", "deployments/operator/samples/deviceplugin_v1_sgxdeviceplugin.yaml")
+
+		if _, err := e2epod.WaitForPodsWithLabelRunningReady(f.ClientSet, ns, labels.Set{"app": "intel-sgx-plugin"}.AsSelector(), 1, timeout); err != nil {
+			framework.Failf("unable to wait for all pods to be running and ready: %v", err)
+		}
```

Comment:
The logs suggest that there's a false positive:
```
Feb  2 20:15:47.856: INFO: Waiting up to 2m0s for at least 1 pods in namespace inteldeviceplugins-system to be running and ready
Feb  2 20:15:47.860: INFO: found 0 pods, waiting for at least 1
Feb  2 20:15:49.974: INFO: 1 pods are not running and ready: [inteldeviceplugins-system/intel-sgx-plugin-59fsz(5ac70873-f6f4-42f9-8c3d-40ce07ee97a7)]
```

I'm not expecting the test to pass because there's another SGX plugin daemonset deployed from the previous step.

Reply:
The log is still the same. Is there an error we don't catch or what might be the reason for that output?